src/azul/service/elasticsearch_service.py

```diff
@@ -295,19 +295,20 @@ def _apply_paging(self, es_search, pagination):
 
         _sort = pagination['sort'] + ".keyword"
         _order = pagination['order']
+        _mode = 'min' if _order == 'asc' else 'max'
```

Comment:
I don't think so. We want to sort by the smallest value in the list in all cases. 

Two documents: `x={"foo": "x", "bar": [1, 4]}` and `y={"foo": "x", "bar": [2,3]}` sorted by field `bar` ascending would give `[x, y]` in that order. With your implementation, the descending sort would give `[x, y]` as well which is broken. The descending sort order should be the inverse of the ascending one.